Dockerfile

```diff
@@ -12,16 +12,19 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
 	liblapack-dev \
 	libopenblas-dev \
 	libssl-dev xz-utils \
-	libclang-3.9-dev clang \
+	libclang-3.9-dev clang libxml2 zlib1g-dev \
 	&& update-alternatives --install "/usr/bin/ld" "ld" "/usr/bin/ld.gold" 20 \
 	&& update-alternatives --install "/usr/bin/ld" "ld" "/usr/bin/ld.bfd" 10
 
 ENV DLANG_VERSION "dmd-nightly"
 ENV DLANG_EXEC "dmd"
 
-RUN curl -fsS -o /tmp/install.sh https://dlang.org/install.sh \
- && bash /tmp/install.sh -p /dlang install ${DLANG_VERSION} \
- && rm -f /dlang/d-keyring.gpg \
+# Download and run the install script
+RUN curl -fsS -o /tmp/install.sh https://dlang.org/install.sh
+RUN bash /tmp/install.sh -p /dlang install ${DLANG_VERSION}
```

Comment:
The idea of doing this in one step is that the temporary files are never saved as sub-images which reduces the storage size of the image.

Reply:
It also makes debugging much harder. The proper way to do it would be to have a build image and a deploy image. Now that I understand better what it's doing I'll try to adapt it in the next PR.